### ISSUE TITLE
Design: Create mockups to indicate sorting by multiple columns

### ISSUE THREAD

**Issue by NilakshiS** [hackforla/tdm-calculator#1999]
[Quick link to bottom of page](https://github.com/hackforla/tdm-calculator/issues/1999#react-issue-comment-composer)

### Overview
Create mockups that could help user identify which columns have an active sort, so that users will be able to see what the sort order is.

### Details
Dev has implemented the ability to sort using multiple columns. . Currently, as per issue #1919 icons are used to indicate which column is being used to sort the table. However, with the new sorting, ~up to 3~ unlimited columns can be used for sorting the data. Using the same icon for all the columns does not provide enough information about the order in which the sorting is applied. Create a design which can appropriately convey this information. Following the previous designs, using icons is suggested however alternate designs can be explored.

- Also, read the details in the linked dev issue to understand how it's implemented.  #1981

<details><summary>Dev Lead's explanation</summary>

The way it is currently implemented, the list defaults to being sorted by dateModified, descending, i.e., most recently modified first. If you select another field (field x, sort direction ascending), then the sort is by dateModfied descending, then field x ascending. If you select a third field y descending, then the sort is by dateModified descending, field x ascending, then field y descending, and so on. If you reset the sort criteria, it goes back to just sorting on dateModified descending. (If you re-apply a field to the sort order that was already part of the sort columns, it is removed from it's initial place in the sequence and added back in to the end of the sequence, since it doesn't make sense to sort by the same column more than once.)

This implementation is pretty intuitive for users as implemented, but there are two aspects that would probably confuse them when you try to explain it in the UI:

First, the part about always having at least one sort order, since they probably think of the list as unsorted until they specify something. I doubt many users realize that the entries are deliberately sorted by date modified, descending, are are pretty unlikely to recognize that they are even sorting by multiple columns when they sorting by one column, then another without resetting the sort in between.
I think the even more confusing part for users is what it means to sort by Column A, then Column B. What this actually does is sort all the rows by Column A, then when you sort by Column B, All the rows with the same value for Column B are grouped together and the groups of similar Column B values are sorted, while within the set of rows with the same value for Column B, they are sorted by Column A values. The last column sorted is considered the "primary" sort column, and users who don't even know that there is a multi-column sort in play would see that the rows appear to be sorted by Column B. This is the way spreadsheet apply multi-column sorting as well. I'm guessing most users would consider this to be sorting by Column B, then Column A instead of A then B, since they would think the Column B values are grouped together and ordered kind of like the top-level of an outline in a document, then within each group of rows with the same value, the rows are sorted by Column A.
</details> 

When you first create the mockup, use the My Projects page, once we have alignment on how that should look, we will have you do mockups for each page that has a sortable table
- My Projects
- Manage Submissions
- Feedback
- Security pages

### Action Items
- [x] Assign yourself to this issue.
- [x] Move the issue from Prioritized Backlog to In Progress.
- [x] In Figma, move a blank page to the Design section.
- [x] Change the Figma page's title to have the format "#Issue Number | Title | Assignee Name".
- [x] Copy contents from the Figma Template page to this issue's Figma page.
- [x] Include a link to this issue's Figma page in the Figma section below.
- [x] Use the Figma page to design solutions for this issue.
- Once this issue is ready to be reviewed:
  - [x] Add a comment tagging the team design lead that [explains the design](https://github.com/hackforla/tdm-calculator/wiki/Design:-Presentations).
  - [x] Move it to the Visual Review column.
  - [x] Add the "ready for Design lead" label.
  - [x] Once you have received feedback from the team lead, implement feedback as necessary.
- [x] Use [presentation structure](https://github.com/hackforla/tdm-calculator/wiki/Design:-Presentations) to review proposed designs with the design director. Implement feedback as necessary.
- [x] Use [presentation structure](https://github.com/hackforla/tdm-calculator/wiki/Design:-Presentations) to review proposed designs with the product, development, and research teams. Implement feedback as necessary.
- [ ] Create a new slide (or edit one if it already exists) in staging and add images and appropriate text
- [ ] Receive design approval from stakeholders. Implement feedback as necessary.
- [ ] Add any annotations to the Figma design that would help the developer with implementation.
- [ ] Create a Ready for Dev section around the final designs on the Figma page.
- [ ] Update the [Figma Design System](https://www.figma.com/design/nD9QK56Mzq7xNSaSUoeGx0/TDM-Calculator?node-id=4041-12655&t=54f7L4Eubz7ppygs-1) with changes resulting from the issue's final design, if applicable.
- [ ] Update the [Style Guide Presentation](https://docs.google.com/presentation/d/1I4q35NL2WW2RpksIyawhHCd8qJnrjme1ZDNIBxu24hQ/), if applicable.
- [ ] Add a comment to the issue that references the final design and adds any annotations that would help the developer with implementation.
- [ ] Create a [development handoff issue](https://github.com/hackforla/tdm-calculator/issues/new?template=development-handoff-issue.md) that references the comment.
- [ ] Move the Figma page to the Development section of the Figma file.
- [ ] Add the development issue's number in parentheses at the beginning of the title of the Figma page.
- [ ] Include the development handoff issue number in the GitHub section below.
- [ ] Reassign any previous assignees to this issue.
- [ ] Close this issue.

#### GitHub
- Old issue: #1919 
- Old issue: #1981 
- New issue to implement: https://github.com/hackforla/tdm-calculator/issues/3225

#### Figma
- [Link to NEW Figma file VERSION 2026-05-11](https://www.figma.com/design/nD9QK56Mzq7xNSaSUoeGx0/TDM-Calculator?node-id=30769-9618&t=MWcTsGcVkbmNqXy4-1)
- [Link to Figma file VERSION 2026-04-24](https://www.figma.com/design/nD9QK56Mzq7xNSaSUoeGx0/TDM-Calculator?node-id=30769-9618&t=YYJNBek3iAyJ8aMi-1)
- [Link to Figma file](https://www.figma.com/design/nD9QK56Mzq7xNSaSUoeGx0/TDM-Calculator?node-id=4041-12658&t=15Z74snhj2Sa8XXw-1)

#### Dev Site
https://tdm-dev.azurewebsites.net

#### Slides
- [Staging Deck](https://docs.google.com/presentation/d/1crZ3IxqA4hAu3qzD7ns93Ieuqjwh6wyEtuX_46cP-fg)
   - ~[Staging Deck, 1999, slide 1](https://docs.google.com/presentation/d/1crZ3IxqA4hAu3qzD7ns93Ieuqjwh6wyEtuX_46cP-fg/edit?slide=id.g3ee8d88321a_1_95)~
   - ~[Staging Deck, 1999, slide 2](https://docs.google.com/presentation/d/1crZ3IxqA4hAu3qzD7ns93Ieuqjwh6wyEtuX_46cP-fg/edit?slide=id.g3ee8d88321a_1_107)~
   - ~[Staging Deck, 1999, slide 3](https://docs.google.com/presentation/d/1crZ3IxqA4hAu3qzD7ns93Ieuqjwh6wyEtuX_46cP-fg/edit?slide=id.g3ee8d88321a_1_22)~
   - ~[Staging Deck, 1999, slide 4](https://docs.google.com/presentation/d/1crZ3IxqA4hAu3qzD7ns93Ieuqjwh6wyEtuX_46cP-fg/edit?slide=id.g3ea47eb6b8a_16_1)~
   - ~[Staging Deck, 1999, slide 5](https://docs.google.com/presentation/d/1crZ3IxqA4hAu3qzD7ns93Ieuqjwh6wyEtuX_46cP-fg/edit?slide=id.g3ee8d88321a_1_48)~
   - ~[Staging Deck, 1999, slide 6](https://docs.google.com/presentation/d/1crZ3IxqA4hAu3qzD7ns93Ieuqjwh6wyEtuX_46cP-fg/edit?slide=id.g3ee8d88321a_1_57)~
   - ~[Staging Deck, 1999, slide 7](https://docs.google.com/presentation/d/1crZ3IxqA4hAu3qzD7ns93Ieuqjwh6wyEtuX_46cP-fg/edit?slide=id.g3ee8d88321a_1_66)~
   - ~[Staging Deck, 1999, slide 8](https://docs.google.com/presentation/d/1crZ3IxqA4hAu3qzD7ns93Ieuqjwh6wyEtuX_46cP-fg/edit?slide=id.g3ee8d88321a_1_83)~
   - ~[Staging Deck, 1999, slide 9](https://docs.google.com/presentation/d/1crZ3IxqA4hAu3qzD7ns93Ieuqjwh6wyEtuX_46cP-fg/edit?slide=id.g3ee8d88321a_1_128)~
   - ~[Staging Deck, 1999, slide 10](https://docs.google.com/presentation/d/1crZ3IxqA4hAu3qzD7ns93Ieuqjwh6wyEtuX_46cP-fg/edit?slide=id.g3ee8d88321a_1_138)~
   - ~[Staging Deck, 1999, slide 11](https://docs.google.com/presentation/d/1crZ3IxqA4hAu3qzD7ns93Ieuqjwh6wyEtuX_46cP-fg/edit?slide=id.g3ee8d88321a_1_149)~
   - ~[Staging Deck, 1999, slide 12](https://docs.google.com/presentation/d/1crZ3IxqA4hAu3qzD7ns93Ieuqjwh6wyEtuX_46cP-fg/edit?slide=id.g3ee8d88321a_1_179)~
- Release Deck
   - [2026-06-09 Release Deck, 1999, slide 1](https://docs.google.com/presentation/d/1PmFeErBaJ5C2pljGF8wG-OgxIUy6NYfXITvlod0KVl4/edit?slide=id.g3ea6cf237f1_0_0)
   - [2026-06-09 Release Deck, 1999, slide 2](https://docs.google.com/presentation/d/1PmFeErBaJ5C2pljGF8wG-OgxIUy6NYfXITvlod0KVl4/edit?slide=id.g3ea6cf237f1_0_8)
   - [2026-06-09 Release Deck, 1999, slide 3](https://docs.google.com/presentation/d/1PmFeErBaJ5C2pljGF8wG-OgxIUy6NYfXITvlod0KVl4/edit?slide=id.g3ea6cf237f1_0_18)
   - [2026-06-09 Release Deck, 1999, slide 4](https://docs.google.com/presentation/d/1PmFeErBaJ5C2pljGF8wG-OgxIUy6NYfXITvlod0KVl4/edit?slide=id.g3ea6cf237f1_0_26)
   - [2026-06-09 Release Deck, 1999, slide 5](https://docs.google.com/presentation/d/1PmFeErBaJ5C2pljGF8wG-OgxIUy6NYfXITvlod0KVl4/edit?slide=id.g3ea6cf237f1_0_36)
   - [2026-06-09 Release Deck, 1999, slide 6](https://docs.google.com/presentation/d/1PmFeErBaJ5C2pljGF8wG-OgxIUy6NYfXITvlod0KVl4/edit?slide=id.g3ea6cf237f1_0_44)
   - [2026-06-09 Release Deck, 1999, slide 7](https://docs.google.com/presentation/d/1PmFeErBaJ5C2pljGF8wG-OgxIUy6NYfXITvlod0KVl4/edit?slide=id.g3ea6cf237f1_0_52)
   - [2026-06-09 Release Deck, 1999, slide 8](https://docs.google.com/presentation/d/1PmFeErBaJ5C2pljGF8wG-OgxIUy6NYfXITvlod0KVl4/edit?slide=id.g3ea6cf237f1_0_63)
   - [2026-06-09 Release Deck, 1999, slide 9](https://docs.google.com/presentation/d/1PmFeErBaJ5C2pljGF8wG-OgxIUy6NYfXITvlod0KVl4/edit?slide=id.g3ea6cf237f1_0_71)
   - [2026-06-09 Release Deck, 1999, slide 10](https://docs.google.com/presentation/d/1PmFeErBaJ5C2pljGF8wG-OgxIUy6NYfXITvlod0KVl4/edit?slide=id.g3ea6cf237f1_0_79)
   - [2026-06-09 Release Deck, 1999, slide 11](https://docs.google.com/presentation/d/1PmFeErBaJ5C2pljGF8wG-OgxIUy6NYfXITvlod0KVl4/edit?slide=id.g3ea6cf237f1_0_88)
   - [2026-06-09 Release Deck, 1999, slide 12](https://docs.google.com/presentation/d/1PmFeErBaJ5C2pljGF8wG-OgxIUy6NYfXITvlod0KVl4/edit?slide=id.g3ea6cf237f1_0_98)

**Comment by elisa-nhsbsa** [comment]
@Rabia2219 @ExperimentsInHonesty 

**My 1st Iteration for the Multiple Column Sorting**

<details>
  <summary>SUGGESTION 1: Adding a Sort Counter on the Overlays</summary>
  
<img width="700" alt="When a user selects a radio button from a Sort option, the counter updates from 0/3 to 1/3." src="https://github.com/user-attachments/assets/6764a1c4-5c49-4a71-b337-34089e02eba7" />

</details>

<details>
  <summary>SUGGESTION 2: Updating the Sort icons with a Counter </summary>
  

<img width="700" alt="When a user adds a Sort to a column the icon updates like this:" src="https://github.com/user-attachments/assets/6f2cd0c7-2c0f-4219-b582-65c2bcaa3090" />

</details>

<details>
  <summary>SUGGESTION 3: Add a ‘Sorted By’ section with Sort Bubbles </summary>
  

<img width="700" alt="Image" src="https://github.com/user-attachments/assets/8979e3f9-be8c-49a7-908f-43b051341e00" />

</details>

<details>
  <summary>SUGGESTION 4: Inform with a Warning Text Tooltip </summary>
  

<img width="700" alt="When a user reached the limit of 3 added sorts and they try to add a 4th from the Overlay, inform them with a warning. The Sort Options of the Overlay will be visibly disabled." src="https://github.com/user-attachments/assets/9600b4fc-334e-4c6e-afb3-f4a23cf42d27" />

</details>

**SORT COUNTER LIMIT EXPERIENCE IN ACTION:**

<details>
  <summary>Demonstration flow </summary>
  

<img width="700" alt="Image" src="https://github.com/user-attachments/assets/9d5fb45d-a3e2-4707-918f-2ea33aa94d3c" />

<img width="700" alt="Image" src="https://github.com/user-attachments/assets/432fa1b0-7c9f-4857-b578-6e31d3722ca8" />

<img width="700" alt="Image" src="https://github.com/user-attachments/assets/7bae3887-48b4-45e7-b184-6becbe1e5613" />

</details>

**Comment by ExperimentsInHonesty** [comment]
Meeting review notes 
- See https://github.com/hackforla/tdm-calculator/issues/2568#issuecomment-3899092748 for current size of icons in context menu, please update
- replace the filled funnel icon with the funnel with underline icon
- make the sorted by boxes the same height as the My Projects, items per page box 
- make the x no larger than the height of the words (use different font or icon if necessary, but keep in mind close x's we use on overlays)
   - see https://www.rei.com/b/blizzard/c/downhill-skis?ir=category%3Adownhill-skis&r=c%3Bb
      - <img width="100" alt="Image" src="https://github.com/user-attachments/assets/70ada8e4-0139-4cd2-b136-8b8eaf6559c2" />

- Add to all team meeting agenda to ask the following
   ```
   - [ ] AP to JD: currently the My Projects overlays disapear when you click anywhere outside of them.  If we wanted the overlays to remain on the screen is that feasible?  e.g., we want to clear sort selections.  

   <details><summary>Example of senario where we would need to clear sort selections while the overlay is open</summary>

   [Add image from last screen in demo] https://github.com/hackforla/tdm-calculator/issues/1999#issuecomment-4314657764


   Additionally we would want our warning box to disapear once one of the sorts has been eliminated, and for the text (3/3 sorts) to be updated to 2/3 sorts)
   </details> 
   ```

**Comment by elisa-nhsbsa** [comment]
@ExperimentsInHonesty @Rabia2219 I updated the image with the 3rd sort bubble

**Comment by AshishDes** [comment]
@entrotech 
- [ ] AP to JD: Currently, the My Projects overlays disappear when you click anywhere outside of them.  If we wanted the overlays to remain on the screen, is that feasible?  e.g., we want to clear sort selections. https://github.com/hackforla/tdm-calculator/issues/1999#issuecomment-4314657764. Additionally, we would want our warning box to disappear once one of the sorts has been eliminated, and for the text (3/3 sorts) to be updated to 2/3 sorts)

<details><summary>Example of a scenario where we would need to clear sort selections while the overlay is open</summary>
<img width="864" alt="Image" src="https://github.com/user-attachments/assets/c8dd7611-bad8-42ab-b75a-922292e5bee4" />

</details>

**Comment by entrotech** [comment]
If you have Sort Badges ("Bubbles") to indicate the sort order, then Suggestion 1 above to have the sort counter in the overlays seems superfluous. 

Currently, you can sort by as many columns as you wish, there is no limitation to sort by only 3 columns, and I don't see a reason to place such an arbitrary limit.  If this is ok, then Suggestion 4 isn't necessary.

We have experimented with leaving the popups open until you press the Apply or Reset button. The problem is that then you can open more than one overlay popup open at a time, which is really awkward and can cause even more space problems than we already have.  I'm not sure how leaving them open would be helpful for what you are trying to do with this issue.  I can't find the description of where you think it would be necessary to leave the popup open.

The way it is currently implemented, the list defaults to being sorted by dateModified, descending, i.e., most recently modified first.  If you select another field (field x, sort direction ascending), then the sort is by dateModfied descending, then field x ascending. If you select a third field y descending, then the sort is by dateModified descending, field x ascending, then field y descending, and so on. If you reset the sort criteria, it goes back to just sorting on dateModified descending. (If you re-apply a field to the sort order that was already part of the sort columns, it is removed from it's initial place in the sequence and added back in to the end of the sequence, since it doesn't make sense to sort by the same column more than once.)

This implementation is pretty intuitive for users as implemented, but there are two aspects that would probably confuse them when you try to explain it in the UI:
- First, the part about always having at least one sort order, since they probably think of the list as unsorted until they specify something. I doubt many users realize that the entries are deliberately sorted by date modified, descending, are are pretty unlikely to recognize that they are even sorting by multiple columns when they sorting by one column, then another without resetting the sort in between.
- I think the even more confusing part for users is what it means to sort by Column A, then Column B. What this actually does is sort all the rows by Column A, then when you sort by Column B, All the rows with the same value for Column B are grouped together and the groups of similar Column B values are sorted, while within the set of rows with the same value for Column B, they are sorted by Column A values.  The last column sorted is considered the "primary" sort column, and users who don't even know that there is a multi-column sort in play would see that the rows appear to be sorted by Column B. This is the way spreadsheet apply multi-column sorting as well.  I'm guessing most users would consider this to be sorting by Column B, then Column A instead of A then B, since they would think the Column B values are grouped together and ordered kind of like the top-level of an outline in a document, then within each group of rows with the same value, the rows are sorted by Column A.

**Comment by ExperimentsInHonesty** [comment]
@AshishDes Please articulate your questions about @entrotech comment here, so that we can save time.

**Comment by AshishDes** [comment]
@ExperimentsInHonesty 

This is what I and @elisa understood from the Dev Lead comment (https://github.com/hackforla/tdm-calculator/issues/1999#issuecomment-4356241364)

- The issue asks for a popover when someone hits 3 sorted columns. But Dev mentioned the system can actually handle unlimited columns, and forcing a 3-column limit just restricts what the tool can already do.

- Keeping the popups open ends up looking really messy with multiple boxes overlapping on the screen.

- Numbering the sorts (1, 2, 3) visually might just confuse people. This is because the system secretly always sorts by "Date Modified" by default, and the way it groups the columns under the hood is backwards from how users might read the numbers.

@elisa-nhsbsa  needs to know if we still want to enforce a 3-column limit, or rethink the design so it matches what the backend can already do?. Please let @elisa-nhsbsa know more if we are going forward with the same design requirements, so that she can continue working on the feedback.

**Comment by ExperimentsInHonesty** [comment]
@elisa-nhsbsa Yes, I thought it was just 3 sorts, not unlimited, glad to have that clarified.  So the mission is the same, figure out  how to indicate sorting by multiple columns that match what the backend can already do.

FYI @AshishDes

**Comment by elisa-nhsbsa** [comment]
@ExperimentsInHonesty @Rabia2219 

I was waiting for this week's meeting to clarify the requirements moving forward as I was uncertain the 3 sort limit was still relevant for this issue.

**In today's meeting we discussed that:**
1. Currently, by default the 'My Projects' table is sorted by the 'Last Saved' date, in descending order. In the mock-ups I am going to create, I will show this visually with a 1 count number before the sort icon for the 'Last Saved' date column and explain why.
2. We are going to remove the Bubbles lozenges and the Sort counter in the overlays. There's no longer going to be a sort limit, we are going to count as many sorts as are being added by the user. My mock-ups will reflect that.
3. The last applied sort will update as the 1st and the second last as the 2nd and so on. This is because the column is sorted in order by the last added sort(s). We see this exact same behaviour with Google Sheets.
4. I need to update the top-left icons above the table and the sort icons in my mock-ups to reflect the current production interface.

**Blockers:** This week has been particularly challenging as I was catching up with some online coursework, AI CLI and Figma MCP experiments, and my part-time work. 

**Availability:** I am going to make time this weekend to properly update the mock-ups.

**Comment by elisa-nhsbsa** [comment]
@ExperimentsInHonesty @Rabia2219 

My 2nd iteration of the multiple columns sort is here.

<details>
  <summary>Updating the Sort icons with a Counter </summary>

<img width="800" alt="Image" src="https://github.com/user-attachments/assets/3159d47f-aeac-4aab-8b91-93084d0626c8" />
</details>

<details>
  <summary>Sort Count Order I</summary>

<img width="800" alt="Image" src="https://github.com/user-attachments/assets/c1009655-05e6-46bd-b05f-e1986f04d1c0" />
</details>

<details>
  <summary>Sort Count Order II</summary>

<img width="800" alt="Image" src="https://github.com/user-attachments/assets/bddd0ad4-8f21-4038-ae42-39283c977228" />
</details>

<details>
  <summary>Full-screen wireframes</summary>

<img width="800" alt="Image" src="https://github.com/user-attachments/assets/f01c1d72-abd2-40c2-a928-47a296aac5f1" />

<img width="800" alt="Image" src="https://github.com/user-attachments/assets/6c343ec3-a869-443a-a943-4485d0d7fa79" />
</details>

[Link to NEW Figma file VERSION 2026-05-11](https://www.figma.com/design/nD9QK56Mzq7xNSaSUoeGx0/TDM-Calculator?node-id=30769-9618&t=MWcTsGcVkbmNqXy4-1)

**Comment by Rabia2219** [comment]
@elisa-nhsbsa 

The designs have been accepted. Thank you for the excellent work. 

Today, Bonnie is meeting with John (the dev lead) to find out if all of the tables have the same sort ability which will guide our next steps.
- My Projects
- Submissions
- Manage Submissions
- Security - Accounts
- Security - Archived Projects
- Security - Archived Accounts

---

Met with John and he says all project tables have the same feature that allows sorting by multiple columns

**Comment by ExperimentsInHonesty** [comment]
## When working on the mockups for the sorting on each page with table.  


### Update Table Headers
Make sure you have the up to date text that should be on the table

- https://github.com/hackforla/tdm-calculator/issues/2869
Table Header changes

- My Projects, Deleted Projects
   - Created On
   - Last Saved
   - Date Deleted (should be changed to "Deleted")
   - Submitted
   - Visibility (remove this column)
- My Projects (add these columns in this order to the right of the DRO column
   - Admin Notes (add this column, visible to admins only)
   - Admin Saved (add this column, visible to admins only)
   - Guidelines Version (add this column, visible to admins only)
   - Admin Saved (add this column, visible to admins only)
- Manage Submissions
   - Submitted (add this column after DRO, visible to admins only)
   - Approval Status (add this column after On Hold, visible to admins only)
   - Admin Notes (add this column after Approval Status, visible to admins only)
   - C of O (add this column after Admin Notes, visible to admins only)
   - Approval Status (TBD this issue will answer that question: https://github.com/hackforla/tdm-calculator/issues/3224)
- Submissions - visable to users only
   - Date Submitted (should be changed to Submitted)
   - Assigned Date (should be changed to Assigned)
   - Invoice Paid Date (should be changed to Invoice Paid)
   - On Hold (add this column after Inovice Paid)
   - Approval Status (add this column after On Hold) (TBD this issue will answer that question: https://github.com/hackforla/tdm-calculator/issues/3224)
   - Replaced by (add column after Approval Status)
   - C of O (add column after Replaced By)



ID | Project Name | Address | Date Submitted | Status Updated | Level | DRO | Staff Assigned | Assigned Date | Invoice Status | Invoice Paid Date | On Hold | Approval Status | CofO Date
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --





### Change that will affect Table Headers and Page names
This change is not ready for you to implement yet, but when this issue is closed we will revisit the changes it asks for on tables related to your mockups
- https://github.com/hackforla/tdm-calculator/issues/3199

**Comment by ExperimentsInHonesty** [comment]
Bonnie to find 
- the latest design for the My Projects > Deleted Projects tab in figma and copy it to user facing screens - then review the above comment for deleted project tab column changes
   - Design issue to add deleted projects tab: #2381
   - [Figma](https://www.figma.com/design/nD9QK56Mzq7xNSaSUoeGx0/TDM-Calculator?node-id=18525-34&p=f&t=EqgLcXVyEjY4RM3h-0)
   - Notes: some of its columns are not visible (but I think they are still there (which you can see by searching for `created on` on that page and you see a ghost outline of the column, so I can't tell if the columns are up  to date but the design of the tab and restore button is, so please figure out if you can reveal the columns and then make sure they match the above notes
   - Design issue to update tables with new columns and other elements: #2445 completed on 2025-11-19
   - [Figma](https://www.figma.com/design/nD9QK56Mzq7xNSaSUoeGx0/TDM-Calculator?node-id=18525-54&p=f&t=SLBadyNScjyvsbBH-0)
   - Notes: this issue is an issue to remake all the table screens with uppdated columns.  Its going to be a good starting point.  Theoreticall all the new mockups should have been copied to the user/admin/security facing screens, but I don't know if that happend.  
- the designs for new columns.  It turns out that you can search for the column name in figma and find a figma page where the column has been added, so you can copy it from there
   - My Projects admin view
   - Manage Submissions view
   - Submissions view

**Comment by ExperimentsInHonesty** [comment]
@elisa-nhsbsa 
- I updated this comment with links and notes.  I think you can work on this now https://github.com/hackforla/tdm-calculator/issues/1999#issuecomment-4461653816

**Comment by elisa-nhsbsa** [comment]
@Rabia2219 I won't attend tonight's call. Other personal issues and commitments are interfering.
I will try complete the designs later. I apologise.

**Comment by elisa-nhsbsa** [comment]
@Rabia2219  @ExperimentsInHonesty 
Below is the design update showing the multiple column sort on the user-facing My Projects - Deleted Projects table:

<details>
  <summary>The Updated Deleted Projects Table</summary>

<img height="750" alt="Image" src="https://github.com/user-attachments/assets/0a17f220-fc94-445d-83ea-ca5a2243c956" />

<img height="750" alt="Image" src="https://github.com/user-attachments/assets/5ee5669f-2b3f-4886-8b2b-69f7d7cbb705" />

<img height="750" alt="Image" src="https://github.com/user-attachments/assets/eb489a9c-e860-4f31-b757-54b3842490c5" />

<img height="750" alt="Image" src="https://github.com/user-attachments/assets/63bb3f6a-2724-4741-8dec-747950581be5" />

<img height="750" alt="Image" src="https://github.com/user-attachments/assets/deca3487-d1c5-4e7a-a28b-7b82c79c22cd" />

<img height="750" alt="Image" src="https://github.com/user-attachments/assets/1a8e387e-99fa-4e38-bfa9-6431ac21d834" />

<img height="750" alt="Image" src="https://github.com/user-attachments/assets/28a1361b-8d38-4013-b6be-d13bc3da6fe9" />

<img height="750" alt="Image" src="https://github.com/user-attachments/assets/7d314a0f-c491-4fa9-92f6-cf7150e05c90" />
</details>

Followed by the update to the Admin view of the My Projects table, showing the multiple sort functionality:

<details>
  <summary>The Updated Admin-Facing My Projects Table </summary>

<img height="750" alt="Image" src="https://github.com/user-attachments/assets/cc221256-4179-40df-bf79-6888213d4a88" />

<img height="750" alt="Image" src="https://github.com/user-attachments/assets/f72c3568-baeb-4c91-b3ed-a4287ece8d94" />

<img height="750" alt="Image" src="https://github.com/user-attachments/assets/e9b7877a-7cdc-4e0b-a299-2853d6d91814" />

</details>

<details>
  <summary>The Updated User-Facing Submissions Table </summary>

<img height="640" alt="Image" src="https://github.com/user-attachments/assets/437d7fc6-fca5-4905-bb80-e611cd48e3f9" />

<img height="640" alt="Image" src="https://github.com/user-attachments/assets/c5798069-09c6-492b-a674-d3248da9f067" />

<img  height="640" alt="Image" src="https://github.com/user-attachments/assets/769e6389-55dc-4e80-8b22-f6f48727372f" />

</details>

<details>
  <summary>The Updated Admin-Facing Manage Submissions Table </summary>

<img  height="682" alt="Image" src="https://github.com/user-attachments/assets/6390d11f-92de-4127-bef0-7951a4d00928" />

<img  height="672" alt="Image" src="https://github.com/user-attachments/assets/8c9ee5d5-cd51-4042-9100-aac6a16702f9" />

<img height="662" alt="Image" src="https://github.com/user-attachments/assets/3a383a48-1540-4336-9f31-f27f28f80d3e" />

</details>

**Comment by Rabia2219** [comment]
@elisa-nhsbsa 
Please check off the boxes under the Action Items for the items that are done

**Comment by Rabia2219** [comment]
@elisa-nhsbsa Please update all the explainer text to reflect
- Sort = Changing the Order
- Filter = Reducing the Results

For example:
In the explainer text 
"3. When the user applies the new sort filter, the 'Created On' filter updates with a counter as the top filter, and 'Last Saved' as the second"
It needs to be changed to
"3. When the user applies the new sort ~filter~, the 'Created On' ~filter~ column updates with a counter as the top ~filter~ sort, and 'Last Saved' as the second"

**Comment by Rabia2219** [comment]
@elisa-nhsbsa  "Approval Status" is being changed to "Submission Status" on Manage Submissions and Submissions. Please update.

**Comment by elisa-nhsbsa** [comment]
@Rabia2219 @ExperimentsInHonesty Here are the updated screens:

<details>
  <summary>Manage Submissions - Admin</summary>

<img height="595" alt="Image" src="https://github.com/user-attachments/assets/ec128676-7cd5-440a-8cc3-0d44f228dd2f" />

<img  height="595" alt="Image" src="https://github.com/user-attachments/assets/62e42e74-d10d-482f-a215-0e0ebf934b4e" />

<img height="595" alt="Image" src="https://github.com/user-attachments/assets/aa4da943-ad4b-4b56-b361-a1ffba170819" />

</details>

<details>
  <summary>Submissions - Users only</summary>

<img height="595" alt="Image" src="https://github.com/user-attachments/assets/e339fa8d-4bcd-4041-b55f-89e5a7d4c3ba" />

<img height="595" alt="Image" src="https://github.com/user-attachments/assets/04e54d44-ec40-43df-863a-ab137608dbea" />

<img height="595" alt="Image" src="https://github.com/user-attachments/assets/14e3d28d-0e50-4634-a5cc-c81c8116fb24" />

</details>

I updated the "Approval Status" to ''Submission Status'', and I took out the word 'filter' when referring to sorting.

[Link to Figma file](https://www.figma.com/design/nD9QK56Mzq7xNSaSUoeGx0/TDM-Calculator?node-id=30769-9602&t=JcIutpqT2cO1i2wb-1)

**Comment by Rabia2219** [comment]
@elisa-nhsbsa Please copy all the [images from this comment](https://github.com/hackforla/tdm-calculator/issues/1999#issuecomment-4575974904), with the final correct explainer text into a new comment on this issue.

**Comment by Rabia2219** [comment]
### Follow up notes from 2026-06-09 stakeholder meeting
- Design approved
- Chose "Status Updated" column to be the default sort for Logged In User>Submissions
- Chose "Submitted" column to be the default sort for Admin User>Manage Submissions

#### Tasks
- [x] Create new Dev issue to use “Status Updated” (5) as the default sort for Logged In User>Submissions
   - #3296 
- [x] Create new Dev issue to use “Submitted” (6) as the default sort for Admin User>Manage Submissions
   - #3297

**Comment by Rabia2219** [comment]
@elisa-nhsbsa As discussed in Friday's meeting (2026-06-12), please copy all the https://github.com/hackforla/tdm-calculator/issues/1999#issuecomment-4575974904, with the final correct explainer text into a new comment on this issue.

Dev issue cannot be worked on until this is completed.

**Comment by elisa-nhsbsa** [comment]
@Rabia2219 @ExperimentsInHonesty Here are the final flows with correct explainer text:

<details>
  <summary>Updating the Column Sorts Icons with Counters</summary>

<img height="986" alt="Image" src="https://github.com/user-attachments/assets/2d8fe71d-6bd0-4322-8da7-4910943940dd" />

</details>

<details>
  <summary>Sort Count Order</summary>

<img height="990" alt="Image" src="https://github.com/user-attachments/assets/88bef668-d4a1-4f93-a582-7a32e34684d9" />

<img height="987" alt="Image" src="https://github.com/user-attachments/assets/d5c734a8-a706-49df-9a08-b93e59086f1c" />

</details>

<details>
  <summary>My projects - Users</summary>

<img height="802" alt="Image" src="https://github.com/user-attachments/assets/5afc682c-116e-4c71-9b68-4b85eb0ee58d" />

<img height="836" alt="Image" src="https://github.com/user-attachments/assets/a0b9e302-4171-4406-8971-0a0408e8119a" />

<img height="830" alt="Image" src="https://github.com/user-attachments/assets/b38bcffe-cc40-4137-a636-47e71897698c" />

</details>

<details>
  <summary>My projects - Users - Deleted</summary>

<img height="1000" alt="Image" src="https://github.com/user-attachments/assets/126728ea-5089-4fac-a737-e246f58b84f3" />

<img height="880" alt="Image" src="https://github.com/user-attachments/assets/535d73c4-3623-436f-ad30-97d081c1d8d2" />

<img height="903" alt="Image" src="https://github.com/user-attachments/assets/25787844-56b8-4520-b1e4-a99af5b27001" />

<img height="871" alt="Image" src="https://github.com/user-attachments/assets/f5be397e-910c-48c2-99ec-d3edae0053a4" />

<img height="892" alt="Image" src="https://github.com/user-attachments/assets/e8192f4d-f24b-4ca5-b862-c69e3f5e4ab2" />

<img height="889" alt="Image" src="https://github.com/user-attachments/assets/cd1485da-6257-4366-a1d2-9d81070e57df" />

<img height="889" alt="Image" src="https://github.com/user-attachments/assets/dfb55621-9669-4cfd-90d7-589aec2a1d9c" />

<img height="1000" alt="Image" src="https://github.com/user-attachments/assets/b770f785-fb9b-437c-8f1d-9258dee52eb6" />

</details>

<details>
  <summary>My projects - Admin</summary>

<img height="971" alt="Image" src="https://github.com/user-attachments/assets/d0bd65ae-bc35-44bf-b817-208e44e2e40e" />

<img height="818" alt="Image" src="https://github.com/user-attachments/assets/555b2308-6c7d-4dca-a2b0-08377ca6d688" />

<img height="849" alt="Image" src="https://github.com/user-attachments/assets/cb005697-11d6-4c32-bf3b-6dcf683a0cd2" />

</details>

<details>
  <summary>Submissions - Users</summary>

<img height="812" alt="Image" src="https://github.com/user-attachments/assets/1079510d-2a44-4a0c-97db-9b74a8cd9d73" />

<img height="640" alt="Image" src="https://github.com/user-attachments/assets/3eb27619-8cd9-4163-81ec-2dcb3ff58d51" />

<img width="1704" height="632" alt="Image" src="https://github.com/user-attachments/assets/a4fd8c0f-3199-4e35-a304-1ca3eda84bbb" />

</details>

<details>
  <summary>Manage Submissions - Admin</summary>

<img height="879" alt="Image" src="https://github.com/user-attachments/assets/89c071f0-82fb-4a07-a72c-c2847866f34b" />

<img height="661" alt="Image" src="https://github.com/user-attachments/assets/fbbb773b-f56d-456d-ba0e-d97585055a31" />

<img height="663" alt="Image" src="https://github.com/user-attachments/assets/0b5da32c-92ae-4df3-9802-f59bd221fa04" />

</details>

[Link to Figma file](https://www.figma.com/design/nD9QK56Mzq7xNSaSUoeGx0/TDM-Calculator?node-id=30769-9602&t=JcIutpqT2cO1i2wb-1)

**Comment by Rabia2219** [comment]
@elisa-nhsbsa Please add showing the sort icon with a number next to it into this area of [Figma: Full Design System, Iconography](https://www.figma.com/design/nD9QK56Mzq7xNSaSUoeGx0/TDM-Calculator?node-id=24025-14586&t=V8eaWRvYkaY7xUOL-0), in addition to what is already there.
<img width="347" height="42" alt="Image" src="https://github.com/user-attachments/assets/dd53284d-7daa-4a3d-a68e-a44fc055286e" />

**Comment by elisa-nhsbsa** [comment]
@Rabia2219 @ExperimentsInHonesty I am running bit late this week with my designs as there's some client commitments and things coming up this week but I will get back with an update as soon as I can.

**Comment by Rabia2219** [comment]
@elisa-nhsbsa Please make the following edits

## Changes Requested to [Your Figma page](https://www.figma.com/design/nD9QK56Mzq7xNSaSUoeGx0/TDM-Calculator?node-id=18525-85&p=f&t=t54sUh78C6BCTvXd-0)


### Submissions - Users

**Submissions - Users Only**

<details><summary>1. The 'Status Updated' with Descending Sort is the default 1st sort on the Submissions page.</summary>


   [ADD IMAGE TO SHOW DEFAULT SORT: NUMBER 1 AND THE DESCENDING SORT ICON ON THE 'Status Updated' COLUMN] 
</details>

<details><summary>2. The user adds a sort to 'ID' column.</summary>

   [REVISE IMAGE TO SHOW NUMBER 1 AND THE DESCENDING SORT ICON ON THE 'Status Updated' COLUMN]

   <img width="2743" height="998" alt="Image" src="https://github.com/user-attachments/assets/450bf920-e456-48f1-b66e-f392fe6b5f89" />
</details>

<details><summary>3. When the user applies the second sort, the table shows sorted by 'ID' 1 and 'Status Updated' 2.</summary>

   [ADD IMAGE TO SHOW NUMBER 1 AND ASCENDING SORT ICON ON THE 'ID' COLUMN AND NUMBER 2 AND THE DESCENDING SORT ICON ON THE 'Status Updated' COLUMN]

</details>
<details><summary>4. The user adds a sort with descending order, to the 'Submitted' column.</summary>

   [REVISE IMAGE TO SHOW NUMBER 1 AND ASCENDING SORT ICON ON THE 'ID' COLUMN AND NUMBER 2 AND THE DESCENDING SORT ICON ON THE 'Status Updated' COLUMN AND KEEP THE OVERLAY]

   <img width="2743" height="998" alt="Image" src="https://github.com/user-attachments/assets/921fe210-edea-48fa-bd02-7f85958aa823" />

</details>
<details><summary>5. When the user applies the new sort, the ‘Submitted' updates with a counter as the top sort, and 'ID' as the second and the 'Status Updated' as the third.</summary>

   [REVISE IMAGE TO ADD THE NUMBER 3 AND THE DESCENDING SORT ICON ON THE 'Status Updated' COLUMN]

   <img width="2743" height="998" alt="Image" src="https://github.com/user-attachments/assets/cbd9c72c-217a-4459-b1f3-2b5f714730ce" />
</details>



### Manage Submissions - Admin

**Manage Submissions - Admin**

<details><summary>1. The ‘Submitted’ with Descending Sort is the default 1st sort on the Manage Submissions page.</summary>

   [ADD IMAGE TO SHOW DEFAULT SORT: NUMBER 1 AND THE DESCENDING SORT ICON ON THE 'Submitted' COLUMN] 

</details>
<details><summary>2. The user adds a sort to 'Created By' column.</summary>
 
   [REVISE IMAGE TO SHOW NUMBER 1 AND THE DESCENDING SORT ICON ON THE 'Submitted' COLUMN AND THE Z-A SORT ORDER ON THE 'Created By' OVERLAY, INDICATED BY A BLUE RADIO BUTTON]

   <img width="2537" height="895" alt="Image" src="https://github.com/user-attachments/assets/029e891f-483e-460b-a109-676b45308636" />

</details>
<details><summary>3. When the user applies the second sort, the table shows sorted by 'Created By' 1 and 'Submitted' 2.</summary>

   [ADD IMAGE TO SHOW NUMBER 1 AND ASCENDING SORT ICON ON THE 'Created By' COLUMN AND NUMBER 2 AND THE DESCENDING SORT ICON ON THE 'Submitted' COLUMN]

</details>
<details><summary>4. The user adds a sort with descending order, to the ‘DRO’ column.</summary>

   [ADD IMAGE TO SHOW NUMBER 1 AND ASCENDING SORT ICON ON THE 'Created By' COLUMN AND NUMBER 2 AND THE DESCENDING SORT ICON ON THE 'Submitted' COLUMN AND THE A-Z SORT ORDER ON THE DRO OVERLAY, INDICATED BY A BLUE RADIO BUTTON]

   <img width="2537" height="895" alt="Image" src="https://github.com/user-attachments/assets/5e94331c-3abe-45ee-b01d-e9f7aa553712" />
</details>

<details><summary>5. When the user applies the new sort, the ‘DRO’ updates with a counter as the top sort, and ‘Created By` as the second and the 'Submitted' as the third.</summary>

   [REVISE IMAGE TO SHOW NUMBER 1 AND DESCENDING SORT ICON ON THE 'DRO' COLUMN AND NUMBER 2 AND THE ASCENDING SORT ICON ON THE 'Created By' AND NUMBER 3 AND THE DESCENDING SORT ICON ON THE 'Submitted' COLUMN]

   <img width="2537" height="895" alt="Image" src="https://github.com/user-attachments/assets/7ebce213-282e-4b9f-a357-e707211f9457" />
</details>

## Changes Requested to the Design System and User/Admin Facing pages in Figma
### [Figma: Full Design System](https://www.figma.com/design/nD9QK56Mzq7xNSaSUoeGx0/TDM-Calculator?node-id=3337-11981&p=f&t=t54sUh78C6BCTvXd-0)
- We revised the overlay section to also include the sort order details.  In the following pages, you will link to that using a line or arrow and this text 
   - See [Design System, Table Headers and Filter/Sort Overlays](https://www.figma.com/design/nD9QK56Mzq7xNSaSUoeGx0/TDM-Calculator?node-id=25560-3127&t=t54sUh78C6BCTvXd-0) for details on how sort is applied.

### [Figma: User-Facing Screens](https://www.figma.com/design/nD9QK56Mzq7xNSaSUoeGx0/TDM-Calculator?node-id=3331-11987&p=f&t=t54sUh78C6BCTvXd-0)
1. [Figma: User-Facing Screens, My Projects](https://www.figma.com/design/nD9QK56Mzq7xNSaSUoeGx0/TDM-Calculator?node-id=18598-576&t=V8eaWRvYkaY7xUOL-0): 
   - Add default sort order number 1 on 'Last Saved' column, next to the icon for the existing screens 
   - Add a link to the Design System page for more details on how second sort works
1. [Figma: User-Facing Screens, Submissions](https://www.figma.com/design/nD9QK56Mzq7xNSaSUoeGx0/TDM-Calculator?node-id=18598-3101&t=V8eaWRvYkaY7xUOL-0): 
   - Add default sort order icon on 'Status Updated' column to existing screen
   - Add number 1 on the 'Status Updated' column to existing screen
   - Add a link to the Design System page for more details on how second sort works

### [Figma: Admin-Facing Screens](https://www.figma.com/design/nD9QK56Mzq7xNSaSUoeGx0/TDM-Calculator?node-id=31408-12453&t=t54sUh78C6BCTvXd-0)
1. [Figma: Admin-Facing Screens, My Projects](https://www.figma.com/design/nD9QK56Mzq7xNSaSUoeGx0/TDM-Calculator?node-id=19038-7187&t=V8eaWRvYkaY7xUOL-0): 
   - Add default sort order number 1 on 'Last Saved' column, next to icon to existing screen
   - Add a link to the Design System page for more details on how second sort works
1. [Figma: Admin-Facing Screens, Manage Submissions](https://www.figma.com/design/nD9QK56Mzq7xNSaSUoeGx0/TDM-Calculator?node-id=31408-10445&t=V8eaWRvYkaY7xUOL-0): 
   - Update default sort order number 1 and icon on 'Submitted' column to existing screen
   - Add a link to the Design System page for more details on how second sort works

**Comment by Rabia2219** [comment]
#### WIP 

### Overview

<details>
<summary>Updating the Column Sorts Icons with Counters</summary>

**Updating the Sort icons with a Counter**

There are 13 table columns. This is how the columns with added sorts look like:

<img width="600" alt="Image" src="https://github.com/user-attachments/assets/35c0a552-5bda-4b9b-a03b-0962a4c20e55" />

When a user adds a Sort to a column, the icon updates like this:

<img width="500" alt="Image" src="https://github.com/user-attachments/assets/8df27ae1-127a-4031-9a20-a839970b340e" />
</details>

<details>
<summary>Sort Count Order - Sorts Only</summary>

**Sort Counter Order I**

1. The ‘Last Saved’ with Descending Sort is the default 1st sort applied to My Projects. Deleted Projects table 

<img width="2077" height="225" alt="Image" src="https://github.com/user-attachments/assets/a56796d2-bf59-4ab2-8ea0-4f91353ba36e" />

2. User goes to add a second sort

<img width="2093" height="564" alt="Image" src="https://github.com/user-attachments/assets/61815a60-6cf4-4f4d-b13d-032299538fd9" />

3. The count order updates in reverse
This is to reflect that the last added sort is at the top in the sort order of the table.

<img width="2064" height="260" alt="Image" src="https://github.com/user-attachments/assets/50bfe622-337b-4a2a-9a74-01f00d691787" />

**Sort Count Order II**

 There is no limit on how many sorts the user can apply.

<img width="2103" height="428" alt="Image" src="https://github.com/user-attachments/assets/a3ec38d4-dd0c-442e-8964-cd108252dfca" />

<img width="2075" height="233" alt="Image" src="https://github.com/user-attachments/assets/ee7414e6-1389-4647-8692-032f175f9178" />

<img width="2062" height="570" alt="Image" src="https://github.com/user-attachments/assets/48294293-33f2-48bc-af63-c53f55d4a0db" />

<img width="2062" height="242" alt="Image" src="https://github.com/user-attachments/assets/2fbfbe34-899f-43d0-a6ff-45f91a3a1d0b" />

</details>

<details>

<summary>Sort Count Order - Sorts and Filter</summary>

**My Projects - Deleted - Users**

<img width="80" alt="Image" src="https://github.com/user-attachments/assets/1458ee43-a9ca-4b5f-aa40-eb133f765cb7" />  1. The ‘Last Saved’ with Descending Sort is the default 1st sort applied to My Projects > Deleted Projects tab

<img width="1808" height="896" alt="Image" src="https://github.com/user-attachments/assets/22b70382-80e5-43ad-89c3-6ebef01596ce" />

2. The user adds a sort with descending order, sorting from Oldest to Newest to the ‘Created On’ date column
<img width="1808" height="896" alt="Image" src="https://github.com/user-attachments/assets/c074f039-4556-411b-8c7e-2c89c89b15ca" />


<img width="80" alt="Image" src="https://github.com/user-attachments/assets/247c26d2-39c4-4d2c-bd8d-8ac70b539843" /> <img width="80" alt="Image" src="https://github.com/user-attachments/assets/595cd515-4fe9-456e-ad51-6b1505826e54" /> 3. When the user applies the new sort, the ‘Created On’ column updates with a counter as the top sort, and ‘Last Saved’ as the second

<img width="1808" height="896" alt="Image" src="https://github.com/user-attachments/assets/a3ba8f20-3b18-4d08-9725-b6f269165d80" />

4. The ‘Date Deleted’ column name has been changed to simply ‘Deleted’. The user adds a sort to this column in ascending order.
<img width="1808" height="896" alt="Image" src="https://github.com/user-attachments/assets/6a2fb0d4-39f2-411a-88e1-bf686780f5fd" />

<img width="80" alt="Image" src="https://github.com/user-attachments/assets/6b1e03ba-4d56-4f0a-810c-7d4da7121b93" /> <img width="80" alt="Image" src="https://github.com/user-attachments/assets/fc940703-efaa-40d3-b464-e80d88a760b9" /> <img width="65" alt="Image" src="https://github.com/user-attachments/assets/29078217-6de8-411c-bd7a-065ff5fd4728" /> 4a. The ‘Deleted’ column updates with a counter as top sort, followed by ‘Created On’ then ‘Last Saved’ as last.

<img width="1808" height="896" alt="Image" src="https://github.com/user-attachments/assets/a2a88874-07b3-4033-ba3a-e1701b0287df" />

5. The user adds a date filter to the ‘Submitted’ column
<img width="1808" height="896" alt="Image" src="https://github.com/user-attachments/assets/78fd66c6-2163-481e-ae0d-acea1d841618" />

6. The user also adds a sort with ascending order from Newest to Oldest date
<img width="1808" height="896" alt="Image" src="https://github.com/user-attachments/assets/67e72bc7-9d8c-4ad3-8c9c-35ed583aaa2e" />

<img width="80" alt="Image" src="https://github.com/user-attachments/assets/8645c917-8e62-48d7-903e-17ff1712c63e" /> <img width="80" alt="Image" src="https://github.com/user-attachments/assets/a0d3d596-7a74-4a29-b125-3e7b233bc43a" /> <img width="65" alt="Image" src="https://github.com/user-attachments/assets/fc205f77-6002-4ad3-bef0-959fc15d54b9" /> <img width="70" alt="Image" src="https://github.com/user-attachments/assets/1e778e58-cfe0-4bcb-b7ca-a3094247b280" /> 7. This now updates the ‘Submitted’ column as the top sort, followed by ‘Deleted’ as 2nd, ‘Created On’ as 3rd, then ‘Last Saved’ as 4th.

<img width="1808" height="896" alt="Image" src="https://github.com/user-attachments/assets/3b740666-c4da-4324-aa57-b4f3ce9aae89" />
</details>


</details>


### Example Pages by Users
<details>
<summary>My projects - Users</summary>
1. The ‘Last Saved’ with Descending Sort is the default 1st sort applied to My Projects > Projects tab

<img height="802" alt="Image" src="https://github.com/user-attachments/assets/5afc682c-116e-4c71-9b68-4b85eb0ee58d" />

2. The user adds a sort to the Visibility column by Sort Visible First

<img height="836" alt="Image" src="https://github.com/user-attachments/assets/a0b9e302-4171-4406-8971-0a0408e8119a" />

3. When the user applies the new sort, the ‘Visibility’ column updates with a counter as the top sort, and ‘Last Saved’ as the second

<img height="830" alt="Image" src="https://github.com/user-attachments/assets/b38bcffe-cc40-4137-a636-47e71897698c" />

</details>

<details>

<summary>My Projects - Deleted - Users</summary>

**My Projects - Deleted - Users**

1. The ‘Last Saved’ with Descending Sort is the default 1st sort applied to My Projects > Deleted Projects tab

<img width="1808" height="896" alt="Image" src="https://github.com/user-attachments/assets/22b70382-80e5-43ad-89c3-6ebef01596ce" />

2. The user adds a sort with descending order, sorting from Oldest to Newest to the ‘Created On’ date column

<img width="1808" height="896" alt="Image" src="https://github.com/user-attachments/assets/c074f039-4556-411b-8c7e-2c89c89b15ca" />

3. When the user applies the new sort, the ‘Created On’ column updates with a counter as the top sort, and ‘Last Saved’ as the second

<img width="1808" height="896" alt="Image" src="https://github.com/user-attachments/assets/a3ba8f20-3b18-4d08-9725-b6f269165d80" />


</details>

<details>
<summary>My projects - Admin</summary>

**My Projects - Admin**

1. The ‘Last Saved’ with Descending Sort is the default 1st sort applied to My Projects table. 
In the Admin view, there are additional columns: ‘Admin Notes’, ‘Admin Saved’ and ‘Guidelines Version’.

<img width="2456" height="999" alt="Image" src="https://github.com/user-attachments/assets/a123c24f-45cb-4085-8d23-d70ddf781ba5" />

2. The user adds a sort with descending order, sorting from Oldest to Newest to the ‘Guidelines Version’ column.

<img width="2456" height="999" alt="Image" src="https://github.com/user-attachments/assets/ccc5426f-acef-4fd8-bb5b-abe85ea28420" />

3. When the user applies the new sort, the ‘Guidelines Version’ updates with a counter as the top sort, and ‘Last Saved’ as the second

<img width="2456" height="999" alt="Image" src="https://github.com/user-attachments/assets/8d3b19f3-bce6-4531-9872-4283db053646" />
</details>

<details>
<summary>Submissions - Users</summary>

**Submissions - Users Only**

1. The 'ID' with Descending Sort is the first sort applied.

<img width="2743" height="998" alt="Image" src="https://github.com/user-attachments/assets/450bf920-e456-48f1-b66e-f392fe6b5f89" />

2. The user adds a sort with descending order, sorting from Oldest to Newest to the ‘Submitted’ date column.

<img width="2743" height="998" alt="Image" src="https://github.com/user-attachments/assets/921fe210-edea-48fa-bd02-7f85958aa823" />

3. When the user applies the new sort, the ‘Submitted’ updates with a counter as the top sort, and ‘ID’ as the second

<img width="2743" height="998" alt="Image" src="https://github.com/user-attachments/assets/cbd9c72c-217a-4459-b1f3-2b5f714730ce" />

</details>

<details>

<summary>Manage Submissions - Admin</summary>

**Manage Submissions - Admin**

1. The 'Created By' is the first sort applied.

<img width="2537" height="895" alt="Image" src="https://github.com/user-attachments/assets/029e891f-483e-460b-a109-676b45308636" />

2. The user adds a sort with descending order, to the ‘DRO’ column.

<img width="2537" height="895" alt="Image" src="https://github.com/user-attachments/assets/5e94331c-3abe-45ee-b01d-e9f7aa553712" />

3. When the user applies the new sort, the ‘DRO’ updates with a counter as the top sort, and ‘Submitted By` as the second.

<img width="2537" height="895" alt="Image" src="https://github.com/user-attachments/assets/7ebce213-282e-4b9f-a357-e707211f9457" />

</details>
---

In the images above, we should always show the default sort icon
Page Name | Default Sort Icon Column | Order
-- | -- | --
My Project | Last Saved | Descending
Manage Submissions | Submitted | Descending
Submissions | Status Updated | Descending

---

When this issue is done, we should have the following
1. User-Facing Screens, My Projects: 
   - Add default sort order number 1 on 'Last Saved' column, next to the icon for the existing screens 
   - Add a link to the Design System page for more details on how second sort works
1. User-Facing Screens, Submissions: 
   - Add default sort order icon on 'Status Updated' column to existing screen
   - Add number 1 on the 'Status Updated' column to existing screen
   - Add a link to the Design System page for more details on how second sort works
1. Admin-Facing Screens, My Projects: 
   - Add default sort order number 1 on 'Last Saved' column, next to icon to existing screen
   - Add a link to the Design System page for more details on how second sort works
1. Admin-Facing Screens, Manage Submissions: 
   - Update default sort order number 1 and icon on 'Submitted' column to existing screen
   - Add a link to the Design System page for more details on how second sort works

#### Use this copy for the link text:
See [Design System, Table Headers and Filter/Sort Overlays](https://www.figma.com/design/nD9QK56Mzq7xNSaSUoeGx0/TDM-Calculator?node-id=25560-3127&t=t54sUh78C6BCTvXd-0) for details on how sort is applied.

**Comment by elisa-nhsbsa** [comment]
@Rabia2219 @ExperimentsInHonesty 

### Submissions - Users

**Submissions - Users Only**

<details>
<summary>1. The 'Status Updated' with Descending Sort is the default 1st sort on the Submissions page.</summary>
<img height="872" alt="Image" src="https://github.com/user-attachments/assets/63741a16-5bd5-4308-95b7-82ec88fc1d20" />
</details>

<details>
<summary>2. The user adds a sort to 'ID' column.</summary>
<img height="701" alt="Image" src="https://github.com/user-attachments/assets/6474253e-93a5-451d-ae31-0b84eabaac67" />
</details>

<details><summary>3. When the user applies the second sort, the table shows sorted by 'ID' 1 and 'Status Updated' 2.</summary>
<img height="705" alt="Image" src="https://github.com/user-attachments/assets/b436d8e5-2e22-4632-941a-4b8fb1c0dea9" />
</details>

<details><summary>4. The user adds a sort with descending order, to the 'Submitted' column.</summary>
<img height="699" alt="Image" src="https://github.com/user-attachments/assets/f3665a51-089f-43b4-aedc-1e2aec9c3058" />
</details>

<details><summary>5. When the user applies the new sort, the ‘Submitted' updates with a counter as the top sort, and 'ID' as the second and the 'Status Updated' as the third.</summary>
<img height="702" alt="Image" src="https://github.com/user-attachments/assets/a9a2236a-9fe3-4805-ba7c-050488ad6824" />
</details>



### Manage Submissions - Admin

**Manage Submissions - Admin**

<details><summary>1. The ‘Submitted’ with Descending Sort is the default 1st sort on the Manage Submissions page.</summary>
<img height="904" alt="Image" src="https://github.com/user-attachments/assets/d6dc0c13-0afd-4fcc-94e0-bd230e64c25d" />
</details>

<details><summary>2. The user adds a sort to 'Created By' column.</summary>
<img height="710" alt="Image" src="https://github.com/user-attachments/assets/21c86dad-4761-489b-9f12-d8355015f7f9" />
</details>

<details><summary>3. When the user applies the second sort, the table shows sorted by 'Created By' 1 and 'Submitted' 2.</summary>
<img height="713" alt="Image" src="https://github.com/user-attachments/assets/2d7a2a74-e073-4874-b943-9c682a43da2f" />
</details>

<details><summary>4. The user adds a sort with descending order, to the ‘DRO’ column.</summary>
<img height="716" alt="Image" src="https://github.com/user-attachments/assets/0b5c3b94-7779-49d4-8b9e-343e6298e0c0" />
</details>

<details><summary>5. When the user applies the new sort, the ‘DRO’ updates with a counter as the top sort, and ‘Created By` as the second and the 'Submitted' as the third.</summary>
<img height="710" alt="Image" src="https://github.com/user-attachments/assets/c79e4924-47c7-4f10-ae41-5545536819e3" />
</details>



### Iconography

<details><summary>Updated the Full Design System Iconography with correct Sort and Filter icons</summary>
<img height="489" alt="Image" src="https://github.com/user-attachments/assets/2bcacb0d-b7d3-4629-9e67-be0c13d3c96b" />
</details>



### User-Facing Screens

**[Figma: User-Facing Screens, My Projects](https://www.figma.com/design/nD9QK56Mzq7xNSaSUoeGx0/TDM-Calculator?node-id=18598-576&t=V8eaWRvYkaY7xUOL-0):**

<details><summary>Updated the User-Facing Screens - My Projects</summary>
<img height="685" alt="Image" src="https://github.com/user-attachments/assets/5890750c-2559-40f4-a6d7-3a7e2a03855d" />
</details>

**[Figma: User-Facing Screens, Submissions](https://www.figma.com/design/nD9QK56Mzq7xNSaSUoeGx0/TDM-Calculator?node-id=18598-3101&t=V8eaWRvYkaY7xUOL-0):**

<details><summary>Updated the User-Facing Screens - Submissions</summary>
<img height="685" alt="Image" src="https://github.com/user-attachments/assets/86be3575-8a22-41ac-a247-80795f2584f3" />
</details>



### Admin-Facing Screens

**[Figma: Admin-Facing Screens, My Projects](https://www.figma.com/design/nD9QK56Mzq7xNSaSUoeGx0/TDM-Calculator?node-id=19038-7187&t=V8eaWRvYkaY7xUOL-0):**

<details><summary>Updated the Admin-Facing Screens - My Projects</summary>

<img height="685" alt="Image" src="https://github.com/user-attachments/assets/6575e472-04b6-4d6f-a4a1-a29588daa71e" />

</details>

**[Figma: Admin-Facing Screens, Manage Submissions](https://www.figma.com/design/nD9QK56Mzq7xNSaSUoeGx0/TDM-Calculator?node-id=31408-10445&t=V8eaWRvYkaY7xUOL-0)**

<details><summary>Updated the Admin-Facing Screens - Manage Submissions</summary>
<img height="685" alt="Image" src="https://github.com/user-attachments/assets/c0a8e922-fcb7-412d-83c0-0254adf0eb73" />
</details>

Added also links to the 3225 in the User and Admin-Facing Screens.

### Question: 

**Do I also need to update this screen in the Admin-Facing page?**
It lacks the 'Submitted' column with the default sort.

<details><summary>Manage Submissions - When Email is clicked, Screen 2</summary>
<img height="685" alt="Image" src="https://github.com/user-attachments/assets/3f2291ae-a1b6-42bb-932e-6d40bd1533e8" />
</details>

**Comment by Rabia2219** [comment]
@elisa-nhsbsa We added the links to the "See [Design System, Table Headers and Filter/Sort Overlays](https://www.figma.com/design/nD9QK56Mzq7xNSaSUoeGx0/TDM-Calculator?node-id=25560-3127&t=t54sUh78C6BCTvXd-0) for details on how sort is applied" to User-Facing Screens, My Projects and Submissions and to Admin-Facing Screens, My Projects and Manage Submissions, as per the [tasks in this comment](https://github.com/hackforla/tdm-calculator/issues/1999#issuecomment-4895580063), but was not done.

### [Figma: User-Facing Screens](https://www.figma.com/design/nD9QK56Mzq7xNSaSUoeGx0/TDM-Calculator?node-id=3331-11987&p=f&t=t54sUh78C6BCTvXd-0)
1. [Figma: User-Facing Screens, My Projects](https://www.figma.com/design/nD9QK56Mzq7xNSaSUoeGx0/TDM-Calculator?node-id=18598-576&t=V8eaWRvYkaY7xUOL-0): 
   - Add a link to the Design System page for more details on how second sort works
<img width="800" alt="Image" src="https://github.com/user-attachments/assets/142e1365-d3e7-4549-9f59-5f17f6f8e263" />

1. [Figma: User-Facing Screens, Submissions](https://www.figma.com/design/nD9QK56Mzq7xNSaSUoeGx0/TDM-Calculator?node-id=18598-3101&t=V8eaWRvYkaY7xUOL-0): 
   - Add a link to the Design System page for more details on how second sort works
<img width="800" alt="Image" src="https://github.com/user-attachments/assets/c8a05232-a360-4b17-9184-27c6d32d17d1" />

### [Figma: Admin-Facing Screens](https://www.figma.com/design/nD9QK56Mzq7xNSaSUoeGx0/TDM-Calculator?node-id=31408-12453&t=t54sUh78C6BCTvXd-0)
1. [Figma: Admin-Facing Screens, My Projects](https://www.figma.com/design/nD9QK56Mzq7xNSaSUoeGx0/TDM-Calculator?node-id=19038-7187&t=V8eaWRvYkaY7xUOL-0): 
   - Add a link to the Design System page for more details on how second sort works
 <img width="800" alt="Image" src="https://github.com/user-attachments/assets/d56229e3-5ef2-4690-80dd-8feed598771a" />

1. [Figma: Admin-Facing Screens, Manage Submissions](https://www.figma.com/design/nD9QK56Mzq7xNSaSUoeGx0/TDM-Calculator?node-id=31408-10445&t=V8eaWRvYkaY7xUOL-0): 
   - Add a link to the Design System page for more details on how second sort works
<img width="800" alt="Image" src="https://github.com/user-attachments/assets/c11d8740-72b2-4f45-997e-220522d85aaf" />

**Comment by Rabia2219** [comment]
@elisa-nhsbsa Please see further requests below:

1.  Can you please explain what you mean by "Added also links to the 3225 in the User and Admin-Facing Screens." ? Do you have a screenshot or link that you can share to show us what you've done?

1. To answer your question below:

> ### Question: 
> 
> **Do I also need to update this screen in the Admin-Facing page?**
> It lacks the 'Submitted' column with the default sort.
> 
> <details><summary>Manage Submissions - When Email is clicked, Screen 2</summary>
> <img height="685" alt="Image" src="https://github.com/user-attachments/assets/3f2291ae-a1b6-42bb-932e-6d40bd1533e8" />
> </details>

This screen does need to be updated as follows:
[REVISE IMAGE BY ADDING THE LATEST TABLE HEADERS ON THE MANAGE SUBMISSIONS PAGE AND ADD THE DEFAULT SORT ICON ON THE 'ID']

**Comment by elisa-nhsbsa** [comment]
> [@elisa-nhsbsa](https://github.com/elisa-nhsbsa) Please see further requests below:
> 
> 1. Can you please explain what you mean by "Added also links to the 3225 in the User and Admin-Facing Screens." ? Do you have a screenshot or link that you can share to show us what you've done?
> 2. To answer your question below:
> 
> > ### Question:
> > **Do I also need to update this screen in the Admin-Facing page?**
> > It lacks the 'Submitted' column with the default sort.
> > Manage Submissions - When Email is clicked, Screen 2
> > <img alt="Image" height="685" src="https://private-user-images.githubusercontent.com/96423084/623321870-3f2291ae-a1b6-42bb-932e-6d40bd1533e8.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3ODUzOTQ5NjksIm5iZiI6MTc4NTM5NDY2OSwicGF0aCI6Ii85NjQyMzA4NC82MjMzMjE4NzAtM2YyMjkxYWUtYTFiNi00MmJiLTkzMmUtNmQ0MGJkMTUzM2U4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNjA3MzAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjYwNzMwVDA2NTc0OVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWYyNmU3ZTNhOGI2NjcyOTc3NmQzNDQzMzYyZTNhNjk0MTdlNWFlMDIzZjZlZDRjMTFiZDc2NzgyZjU4ZjYxM2UmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JnJlc3BvbnNlLWNvbnRlbnQtdHlwZT1pbWFnZSUyRnBuZyJ9.WrGcobeYx6fXX08J36CySSOvOkxKzEuRQPqvg3IAxCE">
> 
> This screen does need to be updated as follows: [REVISE IMAGE BY ADDING THE LATEST TABLE HEADERS ON THE MANAGE SUBMISSIONS PAGE AND ADD THE DEFAULT SORT ICON ON THE 'ID']

@Rabia2219 The links I added can be seen in the last screenshots you shared. Although the links you added highlight it a lot better.

**Comment by Rabia2219** [comment]
@elisa-nhsbsa Thank you for pointing that out. We will keep the ones we added. We look forward to an update with the rest of the tasks in [this comment.](https://github.com/hackforla/tdm-calculator/issues/1999#issuecomment-5025503524)

**Comment by Rabia2219** [comment]
## Final comment for developer

[Figma](https://www.figma.com/design/nD9QK56Mzq7xNSaSUoeGx0/TDM-Calculator?node-id=32037-15158&t=U9naRFPaeUqd4vBe0)


### Sorting 
<details>
  <summary>Updating the Column Sorts Icons with Counters</summary>

<img height="986" alt="Image" src="https://github.com/user-attachments/assets/2d8fe71d-6bd0-4322-8da7-4910943940dd" />

</details>

<details>
  <summary>Sort Count Order</summary>

<img height="990" alt="Image" src="https://github.com/user-attachments/assets/88bef668-d4a1-4f93-a582-7a32e34684d9" />

<img height="987" alt="Image" src="https://github.com/user-attachments/assets/d5c734a8-a706-49df-9a08-b93e59086f1c" />

</details>


#### My projects - Admin

<details><summary>1. The 'Last Saved' with Descending Sort is the default 1st sort on the My Projects - Admin page</summary>
<img height="971" alt="Image" src="https://github.com/user-attachments/assets/d0bd65ae-bc35-44bf-b817-208e44e2e40e" />
</details> 

<details><summary>2. The user adds a sort to 'Guidelines Version' column, in descending order from Oldest to Newest</summary>
<img height="818" alt="Image" src="https://github.com/user-attachments/assets/555b2308-6c7d-4dca-a2b0-08377ca6d688" />
</details> 

<details><summary>3. When the user applies the second sort, the table shows sorted by 'Guidelines Version' as 1 and 'Last Saved' as 2</summary>
<img height="849" alt="Image" src="https://github.com/user-attachments/assets/cb005697-11d6-4c32-bf3b-6dcf683a0cd2" />
</details> 


#### My projects - Users

<details><summary>1. The ‘Last Saved’ with Descending Sort is the default 1st sort applied to My Projects - Users page</summary>
<img height="802" alt="Image" src="https://github.com/user-attachments/assets/5afc682c-116e-4c71-9b68-4b85eb0ee58d" />
</details> 

<details><summary>2. The user adds a sort on 'Visibility' column</summary>
<img height="836" alt="Image" src="https://github.com/user-attachments/assets/a0b9e302-4171-4406-8971-0a0408e8119a" />
</details> 

<details><summary>3. When the user applies the second sort, the table shows sorted by ‘Visibility’ as and ‘Last Saved’ as 2</summary>
<img height="830" alt="Image" src="https://github.com/user-attachments/assets/b38bcffe-cc40-4137-a636-47e71897698c" />
</details> 

#### My projects - Users - Deleted

<details><summary>1. The 'Last Saved' with Descending Sort is the default 1st sort on the My Projects - Users - Deleted tab</summary>
<img height="1000" alt="Image" src="https://github.com/user-attachments/assets/126728ea-5089-4fac-a737-e246f58b84f3" />
</details> 

<details><summary>2. The user adds a sort to 'Created On' column, in descending order from Oldest to Newest</summary>
<img height="880" alt="Image" src="https://github.com/user-attachments/assets/535d73c4-3623-436f-ad30-97d081c1d8d2" />
</details> 

<details><summary>3. When the user applies the second sort, the table shows sorted by ‘Created On as and ‘Last Saved’ as 2</summary>
<img height="903" alt="Image" src="https://github.com/user-attachments/assets/25787844-56b8-4520-b1e4-a99af5b27001" />
</details> 

<details><summary>4. The user adds a sort with ascending order to the ‘Deleted’ column</summary>
<img height="871" alt="Image" src="https://github.com/user-attachments/assets/f5be397e-910c-48c2-99ec-d3edae0053a4" />
</details> 

<details><summary>5. When the user applies the new sort, the ‘Deleted’ updates with a counter as the top sort, and ‘Created On` as the second and the 'Last Saved' as the third.</summary>
<img height="892" alt="Image" src="https://github.com/user-attachments/assets/e8192f4d-f24b-4ca5-b862-c69e3f5e4ab2" />
</details> 

<details><summary>6. The user adds a filter to the 'Submitted' column</summary>
<img height="889" alt="Image" src="https://github.com/user-attachments/assets/cd1485da-6257-4366-a1d2-9d81070e57df" />
</details> 

<details><summary>7. The user adds a sort with ascending order to the 'Submitted' column</summary>
<img height="889" alt="Image" src="https://github.com/user-attachments/assets/dfb55621-9669-4cfd-90d7-589aec2a1d9c" />
</details> 

<details><summary>When the user applies the new sort, the ‘Submitted’ updates with a counter as the top sort, the ‘Deleted` as the second, the 'Created On' as the third and the 'Last Saved' as the fourth.</summary>
<img height="1000" alt="Image" src="https://github.com/user-attachments/assets/b770f785-fb9b-437c-8f1d-9258dee52eb6" />
</details>

**Comment by Rabia2219** [comment]
## Notes for the designer who picks up this issue next

### Progress Made
The designer worked on mockups to indicate sorting on multiple columns. The pages that the mockups were made for are:
- My Projects - Admin
- My Projects - Users
- My Projects - Deleted Tab (Admin and User)
- Submissions - User
- Manage Submission - Admin

The designs were finalized and a dev issue to implement the new designs for the My Projects page was created.

### Each of these pages will be implemented under separate issues because they require other layout changes
   - https://github.com/hackforla/tdm-calculator/issues/2770
   - https://github.com/hackforla/tdm-calculator/issues/2581

### What Remains to be Implemented on this issue
- [x] New design and dev issues to update the design for the Feedback page (to give it a standard header ) were created, but they do not include the multiple sorting mockups. New designs to indicate the multiple sorting need to be created in this issue and added to the User-Facing screens in Figma. If this happens before the dev issue gets assigned, we will add it to the dev issue. If it gets done after, we will make a follow up dev issue.
   - https://github.com/hackforla/tdm-calculator/issues/2838
   - https://github.com/hackforla/tdm-calculator/issues/2863
- [x] The designs for the Security page in the Security-Facing screens need the sort icon and number added to the default sort column `Name`. If this happens before the dev issue gets assigned, we will add it to the dev issue. If it gets done after, we will make a follow up dev issue.
   - https://github.com/hackforla/tdm-calculator/issues/2374
   - https://github.com/hackforla/tdm-calculator/issues/2918
- [ ] New dev issues will need to be made for the following, but are on hold until #3225 is completed
   - Submissions - User
   - Manage Submission - Admin

**Comment by StanzTheDev** [comment]
@Rabia2219 
<details><summary>Updated the design for the Feedback page to include the multiple sorting mockups.</summary>

<img width="400"  alt="Image" src="https://github.com/user-attachments/assets/741b6183-593d-4cba-a605-46fcf295c9d4" />

</details> 

but I’m a little confused about what exactly I’m supposed to do on the Security page. I might need a bit more clarification on that.

**Comment by ExperimentsInHonesty** [comment]
- You should not change the My Projects page, it is fine the way that it is.
- You should copy the sort icon , the number 1 and the explanation box (outlined in green) from the my [User facing Screens page, project page section](https://www.figma.com/design/nD9QK56Mzq7xNSaSUoeGx0/TDM-Calculator?node-id=18598-576&t=bTPr7imkQn7jZCwB-0), and add it to the [Security facing screen, Security pages section](https://www.figma.com/design/nD9QK56Mzq7xNSaSUoeGx0/TDM-Calculator?node-id=27790-1366&t=bTPr7imkQn7jZCwB-0), and apply it to the name column

When you are done, the only thing that will have changed is the [Security facing screen, Security pages section](https://www.figma.com/design/nD9QK56Mzq7xNSaSUoeGx0/TDM-Calculator?node-id=27790-1366&t=bTPr7imkQn7jZCwB-0) which before looked like this

<img width="668" alt="Image" src="https://github.com/user-attachments/assets/1caef8aa-60d3-4c15-a91f-20c019852bfd" />

And now looks like this

<img width="668" height="522" alt="Image" src="https://github.com/user-attachments/assets/3b922a9e-7309-4920-af6f-7ceb85df5dfa" />

**Comment by StanzTheDev** [comment]
@ExperimentsInHonesty 
<details><summary>Updated Security page in the Security-Facing screens to include sort icon and number </summary>

<img width="600" alt="Image" src="https://github.com/user-attachments/assets/7e2215de-0346-4755-b313-c1e7524f050d" />

</details>

**Comment by Rabia2219** [comment]
@StanzTheDev Please add the green box with the link about details on how the header changes on multiple sorts. You can find it on [Figma: Design System: User-Facing Screens, My Projects.](https://www.figma.com/design/nD9QK56Mzq7xNSaSUoeGx0/TDM-Calculator?node-id=18598-576&t=jVOko4CafElYU38e-0)

**Comment by StanzTheDev** [comment]
@Rabia2219 
Done

<img width="319" height="254" alt="Image" src="https://github.com/user-attachments/assets/6c13d734-8b2b-4ad1-b8ec-bb393442f7f4" />

**Comment by ExperimentsInHonesty** [comment]
@StanzTheDev Thank you!!
